src/App.js

```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+//testTask
+import todos from "./TestTodoTasks/testTasks";
+
+//ReactHooks
+import { useState } from "react";
+
+//components
+import TodoCards from "./components/TodoCards/TodoCards";
 
 function App() {
+  const [todoItems, setTodoItems] = useState(todos);
   return (
```

Comment:
good practice is to separate `return` statement and other logic. It helps you to make it clear for everyone.
You also can separate some 'logical' blocks for better readability.

Example:
```
import Something from 'somewhere';
import AnotherThing from 'somewhere-else';

const YourComponent = props => {
  const [thing, setThing] = useState('nothing');
  const [oneMoreThing, setOneMoreThing] = useState('');

  const addSomething = () => {
     //...
  };

  const doAnotherThing = () => {
    //...
  }

  return <div>...</div>
};
```